src/components/InsertIncome.js

```diff
@@ -0,0 +1,305 @@
+import Fade from '@material-ui/core/Fade';
+import { Icon } from 'semantic-ui-react';
+import * as API from '../constants/Api';
+import {UserContext} from './Context';
+
+var newDate = new Date();
+var moment = require('moment');
+var dateIn = moment(newDate);
+var formatedDate=dateIn.format("YYYY-MM-DD");
+
+
+//Adding js styles
+const useStyles = makeStyles( (theme) => ({
+  container: {
+      display: 'flex',    
+      flexWrap: 'wrap',
+      height: "100%",
+  },
+  datepickerx:{
+      width: "35%",
+      marginLeft: "20%",
+      underline: {
+          "&&&:before": {
+          color: "white"
+          },
+          "&&:after": {
+          color: "white"
+          }
+          },
+          '&& .MuiInput-root:hover::before': {
+              borderColor: 'white',
+          },
+          '& .MuiInput-input':{ color: "white"},
+       
+          color: "white",
+          textEmphasisColor: "white",
+          '& .MuiInput-underline:before': {
+          borderBottomColor: "white",
+          '& .MuiInput-underline:after': {
+              borderBottomColor: 'white',
+          },
+          '& .MuiInput-underline:before': {
+              borderBottomColor: 'white',
+          },
+          multilineColor:{
+              color:'white'
+          },
+          '& label.Mui-focused': {
+              color: 'white',
+          },
+          '& label': {
+              color: 'white',
+          },
+          '&:hover fieldset': {
+              borderColor: 'white',
+          },
+          '&.Mui-focused fieldset': {
+              borderColor: 'white',
+          },
+          
+          
+          },
+      },
+  Button:{
+      marginLeft:"1%",
+      marginRight: "1%",
+      background: 'grey',
+      border: 0,
+      borderRadius: 3,
+      color: "white",
+      padding:"3%",
+      width:"90%",
+      marginTop:"7%",
+      textTransform: 'none',
+      backgroundColor: '#37364b',
+      borderColor: '#007bff',
+      '&:hover': {
+      backgroundColor: 'black',
+      borderColor: '#0062cc',
+      },
+      '&:active': {
+      boxShadow: 'none',
+      backgroundColor: 'black',
+      borderColor: '#005cbf',
+      
+      },
+      '&:focus': {
+      boxShadow: 'black',
+      },
+      
+      
+  },
+  textField: {
+    marginLeft:"1%",
+    marginRight: "1%",
+    width:"90%",
+    paddingTop: "1%",
+    marginTop: "4%",
+    color: "white",
+    '& .MuiInput-input':{ color: "white"},
+    multilineColor:{
+    color:'white'
+    },
+    '& label.Mui-focused': {
+    color: 'white',
+    },
+    '& label': {
+    color: 'white',
+    },
+    '& .MuiInput-underline:after': {
+    borderBottomColor: 'white',
+    },
+    '& .MuiInput-underline:before': {
+    borderBottomColor: 'white',
+    },
+    
+    '&:hover fieldset': {
+    borderColor: 'white',
+    },
+    '&.Mui-focused fieldset': {
+    borderColor: 'white',
+    },
+    '&& .MuiInput-root:hover::before': {
+    borderColor: 'white',
+    } 
+    },
```

Comment:
@Sruthyparvathy Follow hyphen delimited string naming convention to name css class name